htcommon/cgi.h

```diff
@@ -24,15 +24,15 @@ class cgi
 	cgi(char *s);
 	~cgi();
 
-	char			*operator [] (char *);
-	char			*get(char *);
-	int			exists(char *);
-	char			*path();
+	const char			*operator [] (const char *);
+	const char			*get(const char *);
+	int							exists(char *);
```

Comment:
You've introduced too much whitespace here. Most project maintainers are very strict about adding whitespace because it adds extra lines to the diff.

Reply:
Should I be using spaces instead of tabs?

Reply:
Oh, good question. The code is kind of mixed I think. Looks like the original developers used tabs. Looks like you did a good job with the whitespace though. I'll have to state a policy on code style. I usually go by this: https://www.gnu.org/prep/standards/standards.html#Formatting